framework/src/meshgenerators/AdvancedExtruderGenerator.C

```diff
@@ -33,21 +34,31 @@
 
 #include <numeric>
 
-registerMooseObject("MooseApp", FancyExtruderGenerator);
+registerMooseObject("MooseApp", AdvancedExtruderGenerator);
+registerMooseObjectRenamed("MooseApp",
+                           FancyExtruderGenerator,
+                           "02/18/2023 24:00",
+                           AdvancedExtruderGenerator);
 
 InputParameters
-FancyExtruderGenerator::validParams()
+AdvancedExtruderGenerator::validParams()
 {
   InputParameters params = MeshGenerator::validParams();
 
   params.addRequiredParam<MeshGeneratorName>("input", "The mesh to extrude");
 
-  params.addClassDescription("Extrudes a 2D mesh into 3D, can have a variable height for each "
-                             "elevation, variable number of layers within each elevation and remap "
-                             "subdomain_ids and element extra integers within each elevation.");
+  params.addClassDescription(
+      "Extrudes a 1D mesh into 2D, or a 2D mesh into 3D, can have a variable height for each "
+      "elevation, variable number of layers within each elevation, variable growth factors of "
+      "axial element sizes within each elevation and remap subdomain_ids, boundary_ids and element "
+      "extra integers within each elevation as well as interface boundaries between neighboring "
+      "elevation layers.");
 
   params.addRequiredParam<std::vector<Real>>("heights", "The height of each elevation");
 
+  params.addRangeCheckedParam<std::vector<Real>>(
+      "biases", "biases>0.0", "The axial growth factor used for mesh biasing for each elevation.");
```

Comment:
we would want a symmetric biasing axially here most probably right?
like finer and finer gpoing up until center, then coarser and coarser?

Reply:
Currently, it can only grow monotonically.

To achieve what you described here, the easiest way is to make two layers of elevations instead of one.

Otherwise, we may need to have something similar to the boundary layer approach in PCCMG.